handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -0,0 +1,310 @@
+# Python DataPlugins
+
+Create a DataPlugin to load, to register, or to search your own custom file formats in LabVIEW or DIAdem, or to index, browse, and find your file formats with SystemLink DataFinder.
+
+You can create DataPlugins using C++, VBS, LabVIEW or Python.

```

Comment:
You can also create DataPlugins using C++, VBS, or LabVIEW.

This way, we start off with the Python DPs, which is the topic of this guide. Then we mention that you can also use other languages, but this is not part of the current guide.